src/SupportMessage/SupportMessage.stories.tsx

```diff
@@ -21,4 +22,12 @@ const supportMessageArgs: SupportMessageProps = {
 
 Default.args = supportMessageArgs
 
+export const WithComponent = Template.bind({})
+
+WithComponent.args = {
+  type: 'info',
+  title: 'A SupportMessage using the Text component',
+  description: <Text tag="p">Some example text</Text>,
+}
```

Comment:
Could you perhaps use your real use cases as an example on Storybook? I think it would make it clearer in which kind of context it is ok to use a custom element as a description because in this case it doesn't bring any value